PowerShell/JumpCloud Module/Public/Users/Get-JCUser.ps1

```diff
@@ -386,24 +386,21 @@ Function Get-JCUser ()
 
                         $Value = ($param.value).replace('*', '')
 
-                        if (($param.Value -match '.+?\*$') -and ($param.Value -match '^\*.+?'))
-                        {
+                        if (($param.Value -match '.+?\*$') -and ($param.Value -match '^\*.+?')) {
                             # Front and back wildcard
-                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "$Value" })
```

Comment:
I think we want the regex filters here too, currently

`Get-JCUser -EmployeeType "* (sales bro)"` | Should return users who's Employee Type is "(anything) (Sales Bro)"
![Screen Shot 2022-07-14 at 3 26 47 PM](https://user-images.githubusercontent.com/54448601/179089033-112914f6-573b-4816-918f-bcdf6d5e2d3d.png)